client/components/data/query-jetpack-onboarding-settings/README.md

```diff
@@ -0,0 +1,44 @@
+Query Jetpack Onboarding Settings
+=================================
+
+`<QueryJetpackOnboardingSettings />` is a React component used in managing network requests for Jetpack Onboarding Settings.
+
+## Usage
+
+Render the component, passing `siteId`. It does not accept any children, nor does it render any elements to the page. You can use it adjacent to other sibling components which make use of the fetched data made available through the global application state.
+
+```jsx
+import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+import QueryJetpackOnboardingSettings from 'components/data/query-jetpack-onboarding-settings';
+import { getJetpackOnboardingSettings } from 'state/selectors';
+
+function MyJetpackOnboardingSettings( { settings, siteId } ) {
+	return (
+		<div>
+			<QueryJetpackOnboardingSettings siteId={ siteId } />
+			{ map( settings, ( value, name ) => (
```

Comment:
Note that some settings (for example `businessAddress`) might contain an object, in which case it won't work to just render them like this.

Reply:
Fair point. I don't want to make the example overly complicated. Let's see :thinking:

Reply:
Added `.toString()`. Doesn't make much sense for an object, but I wanted to keep it really simple.

Reply:
Yeah, it's just an example. `toString()` or just calling concrete keys are both fine for me.